_data/accessibility-tests/character-count.yml

```
@@ -0,0 +1,68 @@
+title: Character count
+component_status: pass
+test_items:
+# General tests
+  - summary: Visible instructions explain the field's input requirements.
+    summary_additional: |
+      The text field has a visible label above it to indicate its purpose. For example, "Enter your full name."
+    test_status: pass
+    test_type: general
+    version_tested: 3.7.1
+    wcag_criterion: 1.3.5
+  - summary: Formatting remains when screen is magnified.
+    summary_additional: |
+      When you zoom in to 200%,
+      the character count keeps its formatting, and sufficient space remains between the character count and the text field above it.
+    test_status: pass
+    test_type: general
+    version_tested: 3.7.1
+    wcag_criterion: 1.4.12
+  - summary: Users can see and hear context about character count errors.
+    summary_additional: |
+      Clear, readable information about character count appears below the text field.
+      When you use a screen reader to tab to the text field, you hear accurate information about how many characters remain for the user to enter.
+    test_status: pass
+    test_type: general
+    version_tested: 3.7.1
+    wcag_criterion: 3.3.2
+# Zoom/screen magnification tests
+  - summary: Content remains visible and functional when screen is magnified.
+    summary_additional: |
+      When you zoom in to 200%,
+      text stays in position inside the text field, and the character count keeps its position below the text field.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.7.1
+    wcag_criterion: 1.4.4
+# Keyboard navigation tests
+  - summary: Field can be accessed with keyboard navigation.
+    summary_additional: Using only a keyboard, you are able to tab into the character count's text input field, type, and tab out of it.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.7.1
+    wcag_criterion: 2.1.1
+# Screen reader tests
+  - summary: Screen reader announces field and character count.
+    summary_additional: |
+      When you tab into the text field with a screen reader,
+      it announces the purpose of the field and the presence of a character count.
+      For example, "This is a text input with a character count."
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.7.1
+    wcag_criterion: 2.4.6
+  - summary: Screen reader announces violations of word limit.
```

Recommending we switch this to "character limit" since the component doesn't count words.
```suggestion
  - summary: Screen reader announces violations of character limit.
```

---

Agreed.

---

I agree, too!

---

Updated "word limit" -> "character limit" in [86b801b](https://github.com/uswds/uswds-site/pull/2607/commits/86b801b05325084d90482c78d0bd881d5d0c3fc3)